app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -374,13 +380,13 @@ public void showVisualizeFragment() {
             mPlaybackToolbar.setNavigationIcon(R.drawable.ic_keyboard_arrow_down_white_24dp);
 
             // move the play button to the middle
-            final float shiftAmnt = (mPlaybackToolbar.getWidth() - playPauseButton.getWidth()) / 2;
+            final float shiftAmnt = (mPlaybackToolbar.getWidth() - mPlayPauseButton.getWidth()) / 2;
             TranslateAnimation animation = new TranslateAnimation(shiftAmnt, 0, 0, 0);
             animation.setDuration(200);
```

Comment:
A proper extraction would be `private static final int MS_ANIMATION_DURATION;` (notice the naming convention -- using MS (representing millisecond) to tell the reader that it is time in milliseconds.